gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```

Comment:
Looks like doing a module purge and reinstalling all of them worked. Going to rebase this change out of the fork.